### ISSUE TITLE
Upload uses "Unspecified" version name (0.2.9)

### ISSUE THREAD

**Issue by brianwernick** [novoda/bintray-release#43]
When trying to upload an aar package to Bintray with 0.2.9 using the following 

```groovy
publish {
    groupId = libraryGroupId
    artifactId = libraryBaseName
    version = '1.0.1'
    autoPublish = false
    dryRun = false
}
```

The version name sent will be "Unspecified".  However using 0.2.7 will correctly use '1.0.1' as the version name.

**Comment by ouchadam** [comment]
thanks for reporting this, out of interest, is the version in the pom correct?

**Comment by brianwernick** [comment]
Yes, the POM lists the correct version '1.0.1'

**Comment by ouchadam** [comment]
seems like our unspecified evaluation is incorrect https://github.com/novoda/bintray-release/blob/master/core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy#L54

we should really be pointing to this https://github.com/gradle/gradle/blob/master/subprojects/core/src/main/groovy/org/gradle/api/Project.java#L223

will fix :+1:

**Comment by ouchadam** [comment]
fixed in `0.2.10`
